Src/Plugins/Security/SEC101_036.MySqlCredentialsValidator.cs

```diff
@@ -61,6 +67,15 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(Dictionary<
             groups.TryGetNonEmptyValue("resource", out FlexMatch resource);
 
             string hostValue = FilteringHelpers.StandardizeLocalhostName(host.Value);
+            string idValue = id.Value;
+
+            // Azure Hosts must be in the form <username>@<hostname> if the username does not contain a host name, we can't connect.
```

Comment:
>Azure Hosts [](http://example.com/codeflow?start=15&length=11)

it's not that azurehost must be. but the userid.
also, let's break the comment in two lines